### ISSUE TITLE
Atlas collapses in walk behavior

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#371]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/371)) by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
When sending foot placements and walk commands using the same foot placement code used in the VRC, the robot goes to take 1 step then collapses.  This happens with atlas and with atals_v3 and atlas_v3_with_v1_intertia.

**Comment by osrf-migration** [comment]
**Original comment by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
OK, so I can get it to take like 3 or 4 steps, but it is certainly not very stable compared to VRC.

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
Are you using the ``keyboard_teleop.launch` control interface?  Can you share more details of the behavior that you are trying out but fail to achieve?

Does same walk behavior still work for you with the old atlas or atlas_sandia_hands model?  If the regression happens only for atlas_v3, it's likely the problem is due to the changes in kinematics of the new atlas_v3 model.  The AtlasSimInterface library was written / designed for the old model kinematics model, and might not tolerate changes in atlas_v3 model.

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------

* set assignee_account_id to "557058:f3968cd3-4910-4384-8349-482a6c7889ec"
* set assignee to "hsu (Bitbucket: [hsu](https://bitbucket.org/hsu), GitHub: [hsu](https://github.com/hsu))"

**Comment by osrf-migration** [comment]
**Original comment by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
We are using the AtlasSimInterface.  I guess the issue was using Atlas v3.  So can we use Atlas v3 with v1 inertia?  Or if walking only supported for the v2 Atlas models?

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
Walking is only officially supported for the v2 Atlas models.  I suppose we can change this issue title to something like "Update AtlasSimInterface Controller Library to Support Atlas V3 Models in Simulation".

And please up-vote this issue to get more attention from controller authors.

**Comment by osrf-migration** [comment]
**Original comment by Brian Gerkey (Bitbucket: [Brian Gerkey](https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/), GitHub: [gerkey](https://github.com/gerkey)).**

----------------------------------------
Another user asked specifically for Manipulate and Step to work.

**Comment by osrf-migration** [comment]
**Original comment by Brian Gerkey (Bitbucket: [Brian Gerkey](https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/), GitHub: [gerkey](https://github.com/gerkey)).**

----------------------------------------

* changed state from "new" to "on hold"

We'll have to wait on BDI to work on the behavior library.

**Comment by osrf-migration** [comment]
**Original comment by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------

* changed state from "on hold" to "closed"

DRC is done people.

**Comment by osrf-migration** [comment]
**Original comment by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
I'm closing this, as I assume by Gazebo 7 this has either been addressed or is no longer something to look into.
